### DISCUSSION TITLE
How to import custom model in tfc_end2end_verification

### DISCUSSION THREAD

**Discussion by B-Willems** [Xilinx/finn#585] [Q&A] [answered]
I want to use the tfc_end2end_verification notebook to verify my finn model but am encountering a few issues.

In this notebook in the second cell, a model gets imported with the command get_test_model_trained("TFC", 1, 1). How can i import my own brevitas export/quantized model in this notebook?

Secondly in the verification of the example network using PyVerilator another weird thing happens. When trying to evaluate, the golden_output is an array of 10 values (as expected), but the output of the rtl simulation is 2. If I look closely at these values, the softmax of golden_output is also 2, so the prediction was successful, however the format in which these are given is not the same. Are there any known fixes for this issue?

An image of the executed cell with added prints can be found below.

Thanks in advance.

![image](https://user-images.githubusercontent.com/15821611/161987118-3e0b6f33-38a4-469a-baf2-c47cf59af7d1.png)

**Comment by auphelia** [comment, marked as answer]
Hi @B-Willems , please check out my comment on your issue: https://github.com/Xilinx/finn/issues/587
